lib/filter/tests/test_filters_regexp.c

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2018 Balabit
+ * Copyright (c) 2013 Gergely Nagy <algernon@balabit.hu>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+#include "filter/filter-expr.h"
+#include "filter/filter-re.h"
+#include "filter/filter-pri.h"
+#include "filter/filter-op.h"
+#include "cfg.h"
+#include "test_filters_common.h"
+
+
+#include <criterion/criterion.h>
+#include <criterion/parameterized.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+TestSuite(filter, .init = setup, .fini = teardown);
+
+typedef struct _FilterParamRegexp
+{
+  const gchar *msg;
+  gint        field;
```

Comment:
We try to avoid such alignment, because when adding new longer variables would cause updating all other unrelated lines.

Reply:
whitespace removed